<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Canvas Eagles - Aircraft Maneuver Log</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,1,0" columnstretch="1,0">
    <item row="0" column="0" rowspan="5">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="date">
      <property name="placeholderText">
       <string>Date</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTabWidget" name="crew_tab">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="turn_controls_grp">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Turn Controls</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="5" column="0" colspan="6">
        <widget class="QPushButton" name="log_turn_btn">
         <property name="text">
          <string>Log Turn</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="maneuver_cmb"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Maneuver:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="6">
        <widget class="QTabWidget" name="gun_tab"/>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="alt_cmb"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Altitude:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTableView" name="turn_log">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="PlanePartDamageTracker" name="engine_grp" native="true"/>
      </item>
      <item>
       <widget class="PlanePartDamageTracker" name="wing_grp" native="true"/>
      </item>
      <item>
       <widget class="PlanePartDamageTracker" name="fusilage_grp" native="true"/>
      </item>
      <item>
       <widget class="PlanePartDamageTracker" name="tail_grp" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="game_menu">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionPlane_Editor"/>
    <addaction name="actionLoad_Planes"/>
    <addaction name="actionReset"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="select_plane_menu">
    <property name="title">
     <string>Select a Plane</string>
    </property>
   </widget>
   <addaction name="game_menu"/>
   <addaction name="select_plane_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLoad_Planes">
   <property name="text">
    <string>Load Planes</string>
   </property>
  </action>
  <action name="actionPlane_Editor">
   <property name="text">
    <string>Create a plane</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlanePartDamageTracker</class>
   <extends>QWidget</extends>
   <header>PlanePartDamageTracker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
